c7n/resources/servicediscovery.py

```diff
@@ -0,0 +1,97 @@
+"""
+Service Discovery Communications
+"""
+from c7n.manager import resources
+from c7n.query import (
+    QueryResourceManager,
+    TypeInfo,
+    DescribeSource,
+    ConfigSource,
+)
+from c7n.tags import universal_augment
+from c7n.filters import ListItemFilter
+
+from c7n.utils import (
+    local_session, type_schema
+)
+
+
+class DescribeServiceDiscoveryNamespace(DescribeSource):
+    # override default describe augment to get tags
+    def augment(self, resources):
+        detailed_resources = super(DescribeServiceDiscoveryNamespace, self).augment(resources)
```

Comment:
in python3, super calls can be simplified to 

`detailed_resources = super().augment(resources)`

as the language will automatically infer self and base class

Reply:
Done